config/configparser/config.go

```diff
@@ -120,33 +116,33 @@ func Load(
 
 	// Start with the service extensions.
 
-	extensions, err := loadExtensions(v.GetStringMap(extensionsKeyName), factories.Extensions)
+	extensions, err := loadExtensions(cast.ToStringMap(v.Get(extensionsKeyName)), factories.Extensions)
```

Comment:
What is the final destination? Are we going to get rid of cast.ToStringMap?

Reply:
I don't think I need it, but will followup. I want to start with a minimal API for `Parser`